index.js

```diff
@@ -0,0 +1,172 @@
+var VirtualKeyboard = {
```

Comment:
I'd suggest using encapsulation, not using window._lastElementFocused, window.VirtualKeyboard, etc.

Reply:
++